### PR TITLE
Partial v3

### PR DESCRIPTION
**Pull request by WcaleNieWolny**

### CODE REVIEW

File: src/bundle/partial.ts [hunk 1]
```diff
@@ -27,49 +44,107 @@ export async function prepareBundlePartialFiles(path: string, snag: LogSnag, org
...
-      file_name: entry.path,
-      s3_path: entry.finalPath,
-      file_hash: entry.hash,
+    else {
+      console.error(errorUpload)
     }
-  })
+  }
 }
+
```
Comment (riderx) [inline src/bundle/partial.ts:109]: remove comment

File: package.json [hunk 2]
```diff
@@ -87,5 +87,13 @@
     "tsconfig-paths": "4.2.0",
     "typescript": "5.5.4",
     "node-dir": "^0.1.17"
+  },
+  "dependencies": {
+    "@types/eventsource": "^1.1.15",
+    "@types/lodash": "^4.17.7",
+    "eventsource": "^2.0.2",
+    "lodash": "^4.17.21",
+    "sse.js": "^2.5.0",
```
Comment (riderx) [inline package.json:96]: remove not used deps

File: package.json [hunk 3]
```diff
@@ -87,5 +87,13 @@
     "tsconfig-paths": "4.2.0",
     "typescript": "5.5.4",
     "node-dir": "^0.1.17"
+  },
+  "dependencies": {
+    "@types/eventsource": "^1.1.15",
+    "@types/lodash": "^4.17.7",
+    "eventsource": "^2.0.2",
+    "lodash": "^4.17.21",
```
Comment (riderx) [inline package.json:95]: do not install lodash, you can just install the exact dep https://www.npmjs.com/package/lodash.chunk

File: src/bundle/upload.ts [hunk 4]
```diff
@@ -541,18 +542,25 @@ export async function uploadBundle(preAppid: string, options: Options, shouldExi
     versionData.storage_provider = 'external'
   }
   else if (zipped) {
-    await uploadBundleToCapgoCloud(supabase, appid, bundle, orgId, zipped, options)
+    // await uploadBundleToCapgoCloud(supabase, appid, bundle, orgId, zipped, options)
```
Comment (riderx) [inline src/bundle/upload.ts:545]: why this is commented?

Comment (WcaleNieWolny) [reply src/bundle/upload.ts:545]: I commented it on accident

**Comment by WcaleNieWolny** [comment]
Closing, rebased at #337

**Pull request closed without merging**